[2025-10-23T20:55:11.322+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-23T20:55:11.346+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v2.get_name manual__2025-10-23T20:55:09.903738+00:00 [queued]>
[2025-10-23T20:55:11.357+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v2.get_name manual__2025-10-23T20:55:09.903738+00:00 [queued]>
[2025-10-23T20:55:11.358+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-10-23T20:55:11.376+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2025-10-23 20:55:09.903738+00:00
[2025-10-23T20:55:11.385+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2370) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-23T20:55:11.388+0000] {standard_task_runner.py:72} INFO - Started process 2372 to run task
[2025-10-23T20:55:11.387+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v2', 'get_name', 'manual__2025-10-23T20:55:09.903738+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpm2i5x1yf']
[2025-10-23T20:55:11.389+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask get_name
[2025-10-23T20:55:11.474+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v2.get_name manual__2025-10-23T20:55:09.903738+00:00 [running]> on host c65f76f8d89c
[2025-10-23T20:55:11.572+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineer' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v2' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-10-23T20:55:09.903738+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-23T20:55:09.903738+00:00'
[2025-10-23T20:55:11.573+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-23T20:55:11.577+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Olovuddin', 'last_name': 'Ramiddinov'}
[2025-10-23T20:55:11.606+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-23T20:55:11.607+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v2, task_id=get_name, run_id=manual__2025-10-23T20:55:09.903738+00:00, execution_date=20251023T205509, start_date=20251023T205511, end_date=20251023T205511
[2025-10-23T20:55:11.647+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-23T20:55:11.655+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
